### ISSUE TITLE
Browser error:  SEC_ERROR_REUSED_ISSUER_AND_SERIAL

### ISSUE THREAD

**Issue by RudyBone** [loft-sh/loft#79]
I am running air-gapped using the loft:1.3.0 image and the 1.2.1 chart.  

Install goes fine, both loft and kiosk pods are running.  I use the port-foward command and go to https://localhost:8080 in my firefox browser and I get the following message:

Error code: SEC_ERROR_REUSED_ISSUER_AND_SERIAL

I suspect this may have something to do with the self signed certs generated but don't know how to proceed.

Also I'd rather use https over an nginx-ingress controller but couldn't seem to get that working either

**Comment by LukasGentele** [comment]
@RudyBone You should be able to use HTTP instead of HTTPS when testing using port-forward. If you want to use an ingress controller, configure TLS in your ingress settings and point the ingress to the service `loft` on port 80 instead of port 443, so that Loft will not handle the SSL handshake but rather the ingress controller will manage the TLS entirely. Here is a sample ingress resource:

```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: loft-ingress
  namespace: loft
  annotations:
    kubernetes.io/ingress.class: nginx
    cert-manager.io/cluster-issuer: lets-encrypt-http-issuer            # in case you are using cert-manager to auto-generate the cert secret
    nginx.ingress.kubernetes.io/proxy-read-timeout: "43200"
    nginx.ingress.kubernetes.io/proxy-send-timeout: "43200"
spec:
  rules:
  - host: loft.yourdomain.tld
    http:
      paths:
      - backend:
          serviceName: loft
          servicePort: 80
        path: /
  tls:
  - hosts:
    - loft.yourdomain.tld
    secretName: tls-loft
```

You can also create this ingress by configuring the appropriate values for the Loft helm chart: https://github.com/loft-sh/loft/blob/master/chart/values.yaml#L32

**Comment by RudyBone** [comment]
Ok, I did the above.  My host is the FQDN of one of my cluster nodes where nginx-ingress controller is running with hostnetwork true on port 443.  If I'm understanding you correctly, I will have TLS from the browser to the nginx controller where TLS will be terminated and the above configuration will call the loft service on port 80 with HTTP.  When I go to my browser I first have to make an exception for the self signed tls-loft cert and then I get BAD Gateway.  In the nginx log, I'm seeing connection refused when it tries to hit http://<loft endpoint ip>:8080/

**Comment by LukasGentele** [comment]
Seems like your Loft instance is not listening on port 80. You can check the service using `kubectl get svc -n loft loft` and the endpoints of it using `kubectl get ep -n loft loft`

**Comment by RudyBone** [comment]
The service is definitely listening on port 80 and the endpoint on 8080.
Still getting connection refused.  I even execed into a pod on my cluster
and tried to curl http://<endpoint ip>:8080 and also got connection refused

**Comment by FabianKramm** [comment]
@RudyBone I think the problem is that the endpoint is listening on port 80 and not 8080, from version 1.3.0 onword loft (the endpoint) listens on port 80 for http and port 443 for https.

**Comment by RudyBone** [comment]
Okay, I switched the targetPort in the Service to 80, the chart version i'm
using is 1.2.1 and it was set to 8080.  Now the login page comes up but
when I enter username: admin and the password I set I get 'error retrieving
access key for user: secret ref is not defined.  I had set admin.create
true in the value.yaml file. Also in the loft pod log, even before I
attempt to login just after it says 'Try creating manager for cluster
loft-cluster' it looks like its trying to call the loft-webhook.loft.svc
webhook, but there is no such service running.  Is it possible that the
chart version I'm using is too out of date to be used with the v1.3.0 image?

**Comment by FabianKramm** [comment]
@RudyBone seems like if you have reinstalled loft multiple times and you did not delete the user in between the installs. To fix this, you can delete the User via `kubectl delete user admin` and then upgrade loft via helm.

Furthermore it makes sense to use at least the chart version 1.3.0 (both chart and image version), since we changed several things there to make air-gapped installation more easy and cross compatability between different chart and image versions is not given.

**Comment by RudyBone** [comment]
I had put this loft eval off during the month of December as I had other
more pressing issues.  I'm going to upgrade everything to the 1.4.1 release
and start over.

**Comment by FabianKramm** [comment]
@RudyBone good idea, let us know if you should encounter any issues

**Comment by FabianKramm** [comment]
@RudyBone this should be fixed in v1.7.0
